### PR TITLE
chore(test): add utils test for validate arweave id

### PR DESCRIPTION
**Pull request by dtfiedler**

### CODE REVIEW

File: spec/ant_spec.lua [hunk 1]
```diff
@@ -6,29 +6,11 @@ local json = require("src.common.json")
...
-	end,
-	id = "test",
-}
-_G.Balances = { [fake_address] = 1 }
-_G.Records = {}
-_G.Controllers = { fake_address }
-_G.Name = "Arweave Name Token"
-_G.Ticker = "ANT"
-_G.Logo = "LOGO"
-_G.Denomination = 1
```
Comment (dtfiedler) [inline spec/ant_spec.lua:21]: Unnecessary duplicate test setup that is in the `describe` block below

File: spec/ant_spec.lua [hunk 2]
```diff
@@ -6,29 +6,11 @@ local json = require("src.common.json")
...
-
-os.clock = function()
-	return 0
-end
```
Comment (dtfiedler) [inline spec/ant_spec.lua:25]: this is done in setup.lua

File: spec/ant_spec.lua [hunk 3]
```diff
@@ -6,29 +6,11 @@ local json = require("src.common.json")
 
 local fake_address = "1111111111111111111111111111111111111111111"
 
-_G.ao = {
-	send = function()
-		return true
-	end,
-	id = "test",
-}
```
Comment (dtfiedler) [inline spec/ant_spec.lua:14]: this is done in setup.lua

File: spec/ant_spec.lua [hunk 4]
```diff
@@ -6,29 +6,11 @@ local json = require("src.common.json")
...
-os.clock = function()
-	return 0
-end
-
 local originalState = {
 	name = "Arweave Name Token",
 	ticker = "ANT",
 	controllers = { fake_address },
-	records = { ["@"] = { transactionId = "test", ttlSeconds = 900 } },
+	records = { ["@"] = { transactionId = fake_address, ttlSeconds = 900 } },
```
Comment (dtfiedler) [inline spec/ant_spec.lua:13]: this was failing now that we've fixed the tx id validation

**Review by atticusofsparta — APPROVED** [review]

**Pull request merged**